jekyll/index.html

```diff
@@ -15,6 +15,33 @@ <h1>Welcome to CircleCI Documentation</h1>
 <hr class="hidden-xs" />
 
 <div class="row">
+  <div class="treatment col-xs-12">
+    <span id="homepage-guide-links"><h2>Examples and Guides</h2><img src="{{ site.baseurl }}/assets/img/compass/new.svg" alt="New"></span>
+    <p>Get started quickly: follow step-by-step <a href="{{site.baseurl}}/2.0/tutorials/">guides</a> or explore a sample app.</p>
+  </div>
+  <div class="treatment col-xs-12 col-sm-6">
+    <a class="no-external-icon col-sm-12" href="{{site.baseurl}}/2.0/language-javascript/">
+      <div class="card col-sm-12">
+        {% capture node-js-card %}
+          {% include snippets/language-card.md lang="Node.JS" anchor="true" %}
+        {% endcapture %}
+        {{ node-js-card | markdownify }}
+      </div>
+    </a>
+  </div>
+  <div class="treatment col-xs-12 col-sm-6">
+    <a class="no-external-icon col-sm-12" href="{{site.baseurl}}/2.0/language-python/">
+      <div class="card col-sm-12">
+        {% capture python-card %}
+          {% include snippets/language-card.md lang="Python" anchor="true" %}
+        {% endcapture %}
+        {{ python-card | markdownify }}
+      </div>
+    </a>
+  </div>
+  <div class="treatment col-xs-12">
+    <hr />
+  </div>
```

Comment:
Could we put this in a seperate .html file and use an include tag, or would that be overkill since we are most likely only using this here for the home page?

Reply:
I can look into it. No guarantee since the 2.0 index page is an MD file. So making the include + variables work for both md and html has been a challenge thus far. There's hacks to make it work that I can see if I can employ.

Reply:
gotcha, most of the hacks Im coming across seems to involve using the `{% raw %}` tag that youre already using